src/scss/skin-modern/components/_subtitleoverlay.scss

```diff
@@ -6,14 +6,22 @@
 
   @include hidden;
 
-  bottom: 2em;
+  align-items: flex-end;
+  display: flex;
   font-size: 1.2em;
-  left: 3em;
+  height: 86%;
+  justify-content: center;
+  margin: auto 10%;
   pointer-events: none;
   position: absolute;
-  right: 3em;
-  text-align: center;
-  transition: bottom $animation-duration-short ease-out;
+  top: 50%;
```

Comment:
Before `ui-subtitle-overlay` had no intrinsic height unless a `subtitle-label` was inserted in it. 

The best solution I could think of to be able to position the subtitles was to make the `ui-subtitles-overlay` div of fixed dimension relative to the player, center it, and then position the `subtitle-label` inside it using `top` and `left`. If I kept the `ui-subtitles-overlay` as before `left` and `top` would not have worked properly.

Then I tried different options to center the `ui-subtitles-overlay` both vertically and horizontally, this is the cleanest I found, but there maybe other solutions.